api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCycleService.java

```diff
@@ -59,14 +60,20 @@ public PaymentCycle createAndSavePaymentCycleForEligibleClaim(Claim claim,
                                                                   LocalDate cycleStartDate,
                                                                   PaymentCycleVoucherEntitlement voucherEntitlement,
                                                                   List<LocalDate> datesOfBirthOfChildren) {
+        LocalDate expectedDeliveryDate = voucherEntitlement.getVouchersForPregnancy() > 0
```

Comment:
Might it be worthwhile this being in a method called something like `getRelevantExpectedDeliveryDate` or `getExpectedDeliveryDateIfRelevant`?

Reply:
Created getExpectedDeliveryDateIfRelevant